packages/analytics-node/test/transport/http.test.ts

```diff
@@ -121,7 +121,7 @@ describe('http transport', () => {
     expect(request).toHaveBeenCalledTimes(1);
   });
 
-  test('should handle unexpected error', async () => {
+  test('should re-throw unexpected error', async () => {
```

Comment:
@falconandy in the latest SDKs we don't throw any errors. We never want the SDK to cause a users app to crash, no matter the state of the SDK.

Reply:
@justin-fiedler 
I believe, transport objects are used by SDK code and errors are caught/processed in callers on SDK level. For example, https://github.com/amplitude/Amplitude-TypeScript/blob/0c03a57239e1048b1730fa58464b125ba9b8b6ba/packages/analytics-core/src/plugins/destination.ts#L168-L172